javascript/modules/client/test/integration/client.test.ts

```diff
@@ -441,31 +507,31 @@ async function createLegacyDao(params: ContextParams) {
     })
 }
 
-export interface ICreateDaoERC20Voting {
+interface ICreateDaoERC20Voting {
```

Comment:
Are these interfaces defined on the testing? Can't they be extracted from the source logic itself?

Reply:
Nope, these interfaces are temporary because they are using the old contracts structure to create a dao, just to test the deposit. Once the new contracts are deployed, I'll delete them

Reply:
Ok, then let's add some clear `// @ TODO` comments, so that it's clear that these types are meant to be removed in short

Reply:
There is already a big comment regarding this